lib/web_of_science/records.rb

```diff
@@ -7,6 +7,8 @@ class Records
     extend Forwardable
     include Enumerable
 
+    delegate logger: :WebOfScience
```

Comment:
OK, keep this but get it out of this commit and put it in an earlier commit where the other logging changes are made too.